### ISSUE TITLE
issue about tr.GraphTransform

### ISSUE THREAD

**Issue by fylinhub** [drorlab/atom3d#50]
Hi, I have an question about da.load_dataset.

I prepared a new 'lmdb' from pdb, and add the label based on the tutorial. However, after reading from da.load_dataset, the 'y' does not show a single value, instead it would show in a form of Dataframe. please see below. Could you advice me how to deal with this?

```
train_dataset = da.load_dataset(PATH_TO_LMDB_OUTPUT, 'lmdb',  transform=tr.GraphTransform(atom_key='atoms', label_key='label'))
for i in train_dataset[0]:
    print(i)
```

('y',      label
0  5.72556)

Thus, when doing for the following command, the batch.y does not show the format of a tensor.

```
batch in train_loader:
    print(batch.y)

```

The way I generated the lmdb is :

```
def add_label(item):
    # Remove the file ending ".pdb" from the ID
    name = item['id'][:-4]
    # Get label data
    label_file = os.path.join(PATH_TO_LABELS_DIR, name+'.csv')
    # Add label data in form of a data frame
    item['label'] = pd.read_csv(label_file)
    return item

## Load dataset from directory of PDB files
dataset = da.load_dataset(PATH_TO_INPUT_DIR, 'pdb', transform=add_label)
# Create LMDB dataset from PDB dataset
da.make_lmdb_dataset(dataset, PATH_TO_LMDB_OUTPUT)

```
If possible, could you give an example of the content in the csv file for add_label? I'd just like to make sure the format is right.
Thank you in advance to looking into this question.

**Comment by fylinhub** [comment]
never mind, I've figured it out.
